pkg/wikidata/wikidata_process_records.go

```diff
@@ -19,17 +19,29 @@
 package wikidata
 
 import (
-	"github.com/richardlehane/siegfried/pkg/wikidata/internal/mappings"
+	"fmt"
+	"log"
 
-	"github.com/ross-spencer/spargo/pkg/spargo"
+	"github.com/richardlehane/siegfried/pkg/wikidata/internal/mappings"
+	"github.com/ross-spencer/wikiprov/pkg/spargo"
 )
 
 // wikidataRecord provides an alias for the mappings.Wikidata object.
 type wikidataRecord = mappings.Wikidata
 
+// getProvenance ...
+func getProvenance(id string, provenance wikiProv) (string, string, error) {
+	for _, value := range provenance {
+		if value.Title == id {
+			return value.Permalink, fmt.Sprintf("%s", value), nil
+		}
+	}
+	return "", "", fmt.Errorf("Provenance not found")
```

Comment:
Consider how to document these `""` return values in the comments or how to make them clearer to the caller. Blank strings look strange without annotation.